shared/src/index.rs

```diff
@@ -150,6 +154,25 @@ impl<T: 'static + KeyValueDB> ChainIndex for ChainKVStore<T> {
     fn delete_block_number(&self, batch: &mut Batch, hash: &H256) {
         batch.delete(COLUMN_INDEX, hash.to_vec());
     }
+
+    fn get_candidate_uncles(&self) -> Vec<H256> {
+        self.get(COLUMN_META, META_CANDIDATE_UNCLES_KEY)
+            .map(|raw| deserialize(&raw[..]).unwrap())
+            .unwrap_or_else(Vec::new)
+    }
+
+    fn insert_candidate_uncle(&self, hash: &H256) -> Result<(), SharedError> {
+        let mut uncles = self.get_candidate_uncles();
+        uncles.push(*hash);
```

Comment:
candidate_uncles increase infinity?

Reply:
> candidate_uncles increase infinity?
Planning to do after this change: https://github.com/nervosnetwork/ckb-dev/issues/47.

How about adding a size limit as a temp solution?